### ISSUE TITLE
Pull access denied for archlinux/base

### ISSUE THREAD

**Issue by andrii-bodnar** [KSXGitHub/github-actions-deploy-aur#15]
Hi, faced with some problems trying to publish the AUR package. The same GH Workflows worked fine a few weeks ago.

Workflow:

```yml
name: publish-aur

on:
  push:
    tags:
      - '*'

jobs:
  aur-publish:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2

      - name: Publish AUR package
        uses: KSXGitHub/github-actions-deploy-aur@v2.2.3
        with:
          pkgname: crowdin-cli
          pkgbuild: ./pkgbuild/PKGBUILD
          commit_username: ${{ secrets.AUR_USERNAME }}
          commit_email: ${{ secrets.AUR_EMAIL }}
          ssh_private_key: ${{ secrets.AUR_SSH_PRIVATE_KEY }}
          commit_message: Update AUR package
          ssh_keyscan_types: rsa,dsa,ecdsa,ed25519
```

Action logs:

```console
2021-04-30T08:32:29.8974463Z Prepare workflow directory
2021-04-30T08:32:29.9662482Z Prepare all required actions
2021-04-30T08:32:29.9677289Z Getting action download info
2021-04-30T08:32:30.4233881Z Download action repository 'actions/checkout@v2'
2021-04-30T08:32:32.2525357Z Download action repository 'KSXGitHub/github-actions-deploy-aur@v2.2.3'
2021-04-30T08:32:32.7895932Z ##[group]Build container for action use: '/home/runner/work/_actions/KSXGitHub/github-actions-deploy-aur/v2.2.3/Dockerfile'.
2021-04-30T08:32:32.7980113Z ##[command]/usr/bin/docker build -t 8a33c1:d4a793bdb67741cabac3a7c3694369c7 -f "/home/runner/work/_actions/KSXGitHub/github-actions-deploy-aur/v2.2.3/Dockerfile" "/home/runner/work/_actions/KSXGitHub/github-actions-deploy-aur/v2.2.3"
2021-04-30T08:32:33.0699625Z Sending build context to Docker daemon  17.92kB
2021-04-30T08:32:33.0705274Z 
2021-04-30T08:32:33.1271431Z Step 1/7 : FROM archlinux/base
2021-04-30T08:32:34.4111151Z pull access denied for archlinux/base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
2021-04-30T08:32:34.4132457Z ##[warning]Docker build failed with exit code 1, back off 4.587 seconds before retry.
2021-04-30T08:32:39.0035055Z ##[command]/usr/bin/docker build -t 8a33c1:d4a793bdb67741cabac3a7c3694369c7 -f "/home/runner/work/_actions/KSXGitHub/github-actions-deploy-aur/v2.2.3/Dockerfile" "/home/runner/work/_actions/KSXGitHub/github-actions-deploy-aur/v2.2.3"
2021-04-30T08:32:39.0509679Z Sending build context to Docker daemon  17.92kB
2021-04-30T08:32:39.0519804Z 
2021-04-30T08:32:39.1092820Z Step 1/7 : FROM archlinux/base
2021-04-30T08:32:40.3475306Z pull access denied for archlinux/base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
2021-04-30T08:32:40.3507723Z ##[warning]Docker build failed with exit code 1, back off 6.581 seconds before retry.
2021-04-30T08:32:46.9355422Z ##[command]/usr/bin/docker build -t 8a33c1:d4a793bdb67741cabac3a7c3694369c7 -f "/home/runner/work/_actions/KSXGitHub/github-actions-deploy-aur/v2.2.3/Dockerfile" "/home/runner/work/_actions/KSXGitHub/github-actions-deploy-aur/v2.2.3"
2021-04-30T08:32:46.9870657Z Sending build context to Docker daemon  17.92kB
2021-04-30T08:32:46.9878502Z 
2021-04-30T08:32:47.0477518Z Step 1/7 : FROM archlinux/base
2021-04-30T08:32:48.2820201Z pull access denied for archlinux/base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
2021-04-30T08:32:48.2830749Z ##[endgroup]
2021-04-30T08:32:48.2952499Z ##[error]Docker build failed with exit code 1
2021-04-30T08:32:48.3009219Z Cleaning up orphan processes
```

Could you please suggest the possible reason of this problem?

**Comment by KSXGitHub** [comment]
I don't know about this myself. This seems to be a GitHub Action or Docker problem.

**Comment by Herschel** [comment]
Hit the same issue. Was working on 4/28 and earlier, failed starting on 4/29. I think the archlinux Docker images have been reorganized, as `archlinux/base` no longer exists. The Dockerfile should perhaps instead use `archlinux/archlinux:base`.

**Comment by KSXGitHub** [comment]
Can you try KSXGitHub/github-actions-deploy-aur@986ddd34dff0100127f95aee3590b4c303617c2f? If it works, then I will merge.

**Comment by Herschel** [comment]
Yep, can confirm that commit works. Thank you!

**Comment by KSXGitHub** [comment]
Released [v2.2.4](https://github.com/KSXGitHub/github-actions-deploy-aur/releases/tag/v2.2.4).
